cves/kernel/CVE-2016-2545.yml

```diff
@@ -75,7 +90,8 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs:
+  - https://bugzilla.redhat.com/show_bug.cgi?id=1311560
```

Comment:
I do not think this bugzilla entry is relevant as I believe the entry only wants kernel bugzilla entries